tests/scripts/thread-cert/pktverify/packet_filter.py

```diff
@@ -462,6 +462,9 @@ def filter_ipv6_src_dst(self, src_addr, dst_addr, **kwargs):
         assert isinstance(dst_addr, (str, Ipv6Addr))
         return self.filter(lambda p: p.ipv6.src == src_addr and p.ipv6.dst == dst_addr, **kwargs)
 
+    def filter_RLARA(self, **kwargs):
```

Comment:
```suggestion
    def filter_RLARMA(self, **kwargs):
```
The convention is to use `MA` at the end of method name.